code/__HELPERS/global_lists.dm

```diff
@@ -33,30 +33,36 @@
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/moth_markings, GLOB.moth_markings_list)
 	init_sprite_accessory_subtypes(/datum/sprite_accessory/pod_hair, GLOB.pod_hair_list)
 
-	//Species
+/// Inits GLOB.species_list. Not using GLOBAL_LIST_INIT b/c it depends on GLOB.string_lists
+/proc/init_species_list()
 	for(var/spath in subtypesof(/datum/species))
 		var/datum/species/S = new spath()
 		GLOB.species_list[S.id] = spath
 	sort_list(GLOB.species_list, GLOBAL_PROC_REF(cmp_typepaths_asc))
 
-	//Surgeries
+/// Inits GLOB.surgeries
+/proc/init_surgeries()
```

Comment:
`return .` at the end here -- https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-

Reply:
Also this should almost certainly be with some surgeries file rather than this god file

Reply:
Yeah, but I'm just leaving it where it is for now